pentagon/migration/migrations/migration_1_2_0.py

```diff
@@ -0,0 +1,163 @@
+
+import pentagon
+from pentagon import migration
+from collections import OrderedDict
+from pentagon.migration import *
+
+
+class Migration(migration.Migration):
+    _starting_version = '1.2.0'
+    _ending_version = '2.0.0'
+
+    def run(self):
+
+        # Create inventory directory if it exists
+        inventory_dir = '{}/inventory'.format(self._infrastructure_repository)
+        if not os.path.isdir(inventory_dir):
+            os.mkdir(inventory_dir)
+
+        # Move default
+        if os.path.exists('{}/default'.format(self._infrastructure_repository)):
+            self.move('default', "inventory/default")
+
+        self.delete('config/local/env-vars.sh')
```

Comment:
`env-vars.sh` is no longer required. It's been replaces with `yaml_source` that lives in the package instead of in the infrastructure repo.

As far as why delete before moving the directory, no real reason other than that's the way it fell out of my brain. It could be deleted after then move as well.